webapp/portlet/src/main/webapp/common/js/SuggesterService.js

```diff
@@ -56,7 +56,7 @@ function searchUsers(filter, items, typeOfRelation, searchOptions) {
               providerId: 'organization',
               profile: {
                 fullName: item.name || item.text,
-                avatarUrl: item.avatar || item.avatarUrl || `/portal/rest/v1/social/users/${username}/avatar`,
+                avatarUrl: item.avatarUrl ? item.avatarUrl : `/portal/rest/v1/social/users/${username}/avatar`,
```

Comment:
Why changing this expression ? doesn't it work for bringing **Users** avatars ?
As you can see, the method name is `searchUsers`. For spaces, there is another method called `searchSpaces`

Reply:
I change it because in event creation when adding a participants we cannot retrieve the avatar of user, all time we retrieve the default avatar.